doc/articles/features/customizing-uiapplicationdelegate.md

```diff
@@ -0,0 +1,55 @@
+---
+uid: Uno.Features.CustomizingUIApplicationDelegate
+---
+
+# Customizing the `UIApplicationDelegate` on iOS
+
+Uno Platform provides the ability to provide custom behavior for `UIApplicationDelegate` on iOS in case of both native- and Skia-based rendering.
```

Comment:
remove `-` after native

Reply:
This should be grammatically correct as far as I know, should be a "suspended hyphen" https://www.quora.com/What-is-a-suspended-hyphen